spec/services/authn_context_resolver_spec.rb

```diff
@@ -148,8 +148,8 @@
     end
   end
 
-  context 'when users uses an acr_values param' do
-    context 'no service provider' do
+  context 'when resolving acr_values' do
+    context 'with no service provider' do
```

Comment:
When does this case arise?

Reply:
if you trace the `self.no_sp_result` method back in the Vot::Parser, you can find this PR that explains it better https://github.com/18F/identity-idp/pull/10092 but basically some requests don't include service provider details.